portfolio/src/main/webapp/index.html

```diff
@@ -39,14 +39,23 @@ <h1>Welcome To Jason's Portfolio</h1>
           <div id="random-image-container"></div>
         </div>
       </div>
+      <button onclick="openQuoteModal()" id = "Quotes">Favorite Quotes</button>
+      <div id="myQuoteModal" class="modal">
+        <div class="modal-content">
+          <span onclick="closeQuoteModal()" class="close">&times;</span>
+          <p>Check out some of my favorite quotes.</p>
+          <button onclick="randomizeQuote()">Click Here</button><br><br>
+          <div id="random-quote-container"></div>
+        </div>
+      </div>
     <button onclick="openConnectModal()" id = "Connect">Connect With Me</button>
     <div id="myConnectModal" class="modal">
       <div class="modal-content">
         <span onclick="closeConnectModal()" class="close">&times;</span>
         <p>Feel free to connect with me on LinkedIn and GitHub.</p>
         <a href="https://www.linkedin.com/in/jasonfewjr/" target="_blank">LinkedIn</a>
         <a href="https://github.com/jfewii"  target="_blank"> GitHub </a>
-      <div id="random-image-container"></div>
+    </div>
     </div>
     </div>
     </div>
```

Comment:
nit: seems like the indentation for these </div> brackets is off. Could you fix?